backend/coins/eth/account.go

```diff
@@ -42,6 +42,8 @@ type Account struct {
 	getSigningConfiguration func() (*signing.Configuration, error)
 	signingConfiguration    *signing.Configuration
 	keystores               keystore.Keystores
+	offline                 bool
+	onEvent                 func(Event)
```

Comment:
You could have used the observable package.

Reply:
Using the same as btc/account.go, so we can refactor both at once.